UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Controllers/ExchangeSetControllerTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using UKHO.ExchangeSetService.API.Controllers;
+using UKHO.ExchangeSetService.API.Services;
+using UKHO.ExchangeSetService.Common.Models;
+using UKHO.ExchangeSetService.Common.Models.Response;
+using UKHO.ExchangeSetService.Common.Models.V2.Request;
+
+namespace UKHO.ExchangeSetService.API.UnitTests.Controllers
+{
+    [TestFixture]
+    public class ExchangeSetControllerTests
```

Comment:
Can we add Unit test for passing null productIdentifier as well please.

Reply:
Added